package.json

```diff
@@ -72,7 +72,7 @@
   "lint-staged": {
     "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
       "prettier --write",
-      "npm test -- --watchAll=false"
+      "npm test -- --watchAll=false --passWithNoTests"
```

Comment:
Why do we need to add this?

Reply:
@hackerwins If this option is not used, testcase must exist for the file being modified .. 😭